README.md

```diff
@@ -427,6 +428,50 @@ var signIn = new OktaSignIn(
 );
 ```
 
+### showSignIn
+
+> **Note**:
+ The showSignIn method is backward compatible. You can use it with both Okta Identity Engine (as of Widget v5.5.0) and Okta Classic Engine.
+
+Recommended for most use cases. Receives an interaction code, either by Promise or redirect, depending on [sign-in policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/procedures/set-up-default-app-redirect.htm). Renders the widget to the DOM to prompt the user to sign in.
+
+On successful [authentication](https://developer.okta.com/docs/guides/oie-intro/#deployment-models), one of the following happens:
+* The Promise is resolved to an object containing OAuth tokens.
+* The browser is redirected to Okta with information to begin a new session. Okta servers process the information and then redirect back to your app's `redirectUri`.
+  * If successful, an authorization code appears in the URL as the `code` query parameter.
+  * If unsuccessful, an `error` query parameter appears in the URL.
+
+The following properties are available when using the `showSignIn` method:
+* `baseUrl` (required) - The base URL for your domain. Can be any Okta org. See [Basic config options](#basic-config-options).
```

Comment:
These constructor options are already documented. We should document here which options can be passed to the method. These are the same options as `showSignInToGetTokens` (el, redirectUri, clientId, scopes)